nodebuilder/tests/swamp/swamp.go

```diff
@@ -224,11 +237,9 @@ func (s *Swamp) NewNodeWithStore(
 ) *nodebuilder.Node {
 	var n *nodebuilder.Node
 
-	ks, err := store.Keystore()
-	require.NoError(s.t, err)
-
+	signer := apptypes.NewKeyringSigner(s.ClientContext.Keyring, s.Accounts[0], s.ClientContext.ChainID)
```

Comment:
Would be nice to rotate accounts for diff nodes, but also good

Reply:
I think it's not necessary to have such functionality because for test purposes we should only be able to submit txs and it does not matter how many accounts will be used